### PR TITLE
Minia-1.6088-goolf-1.4.10.eb

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

**Comment by fgeorgatos** [comment]
Test report by @fgeorgatos
**SUCCESS**
Build succeeded for 1 out of 1
Linux debian 6.0.10, Intel(R) Xeon(R) CPU           L5640  @ 2.27GHz, Python 2.6.6
See https://gist.github.com/4d6bfdf200e5b2990e15 for a full test report.

File: easybuild/easyconfigs/m/Minia/Minia-1.6088-goolf-1.4.10.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+files_to_copy = [(["minia"], "bin"), "LICENSE", "README", "manual", "test"]
+        
+sanity_check_paths = {
+    'files': ["bin/minia"],
+    'dirs': ["manual"],
+}
+
+moduleclass = 'bio'
+
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/m/Minia/Minia-1.6088-goolf-1.4.10.eb]: remove extra line; looks good; @boegel: test/merge?

Comment (fgeorgatos) [inline easybuild/easyconfigs/m/Minia/Minia-1.6088-goolf-1.4.10.eb]: lgtm , minor changes needed here, put this in REVIEW plz!

**Comment by migueldiascosta** [comment]
@pescobar, this PR is being closed for the following reason(s): no activity for > 6 months.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**